rviz_visual_tests/docs/README.md

```diff
@@ -0,0 +1,89 @@
+# A quick start guide: how to write and run RViz visual tests
+A deeper explanation on what this testing framework is about and on how it works can be found in
+[this document](documentation.md).
```

Comment:
Please use the "one line per sentence rule", see: https://github.com/ros2/ros2/wiki/Developer-Guide#markdown

It helps keep diff's small, and prevents you from spending lots of time manually wrapping lines. We require wrapping of lines in doc blocks because code linters usually can't differentiate between source code and doc blocks.